docs/csharp/programming-guide/classes-and-structs/ref-returns.md

```diff
@@ -29,16 +29,19 @@ In addition, reference return values are not allowed on async methods. An asynch
  
 ## Defining a ref return value
 
-You define a ref return value by adding the [ref](../../language-reference/keywords/ref.md) keyword to the return type of the method signature. For example, the following signature indicates that the `GetContactInformation` property returns a reference to a `Person` object to the caller:
+A method that returns a *reference return value* must satisfy the following two conditions:
 
-```csharp
-public ref Person GetContactInformation(string fname, string lname);
-```
+- the method signature includes the [ref](../../language-reference/keywords/ref.md) keyword in front of the return type,
```

Comment:
I think this would be better shown as an unordered list.

Reply:
Isn't it already unordered @BillWagner? Or it should be numbered?

Reply:
Ahh. :eyes: weren't seeing the '-' earlier. 

Then, just two very small nits: Each should be a sentence, so period after "return type" in the first, instead of a comma, and capitalize "The" and "Each".

Reply:
Updated! Thank you for review.